README.md

```diff
@@ -66,13 +66,21 @@ DesignSafe [MkDocs](https://mkdocs.readthedocs.io/) documentation with **customi
 0. Have Docker installed.\
     <sup>We recommend doing so via [Docker-Desktop](https://www.docker.com/products/docker-desktop).</sup>
 1. Navigate into your clone of this repository.
-2. Start the Docker container to serve the docs.
-    ```shell
+2. Start the Docker container to serve the docs. 
+
+   Linux or Mac (macOS) user:
+   ```shell
     make build
     make start
 
     ```
-3. Open the website at the URL provided e.g.
+   Windows user:
+   ```shell
+    docker-compose -f ./docker-compose.yml build
+    docker-compose -f docker-compose.yml up
```

Comment:
**Great!** Either of these suggestions would be more consistent with the path.

```suggestion
    docker-compose -f ./docker-compose.yml build
    docker-compose -f ./docker-compose.yml up
```

```suggestion
    docker-compose -f docker-compose.yml build
    docker-compose -f docker-compose.yml up
```

Reply:
Tested both, and both work!